app/views/v1/subdomain_util.py

```diff
@@ -0,0 +1,109 @@
+from typing import Set, Tuple
+
+import json
+
+import flask_jwt_extended
+from flask import request, url_for
+
+import app.utils.authentication_utils as authentication_utils
+import app.actions as actions
+import app.db_models as db_models
+
+from . import bp
+from app.utils.ct_search import get_subdomains_from_ct
+from app.actions.add_targets import add_targets
+
+from time import time
+
+BATCH_SIZE = 10  # set appropriate amount
+RESCAN_INTERVAL = 62400  # to check target once a day
+
+
+@bp.route("/rescan_subdomains", methods=["GET"])  # is ok?
+@flask_jwt_extended.jwt_required
+def api_cron_rescan_subdomains() -> None:
+    rescan_subdomains()
```

Comment:
Check of JWT token is token needed if we want to expose this to users. For backend stuff there are other mechanisms.

Check `sslyze_scan_due_targets_multiple_batches`.
https://github.com/TLSInventory/backend/blob/ca3d4309b4b6e7571ddeec70cdca610f1ddbc416/app/views/v1/sensor_collector.py#L47